modules/ocf_postgres/manifests/init.pp

```diff
@@ -32,6 +32,22 @@
       };
   }
 
+  # Postgres expects the letsencrypt certs to be owned by root
+  # The renewal scrypt expects the letsencrypt certs to be owned by ocfletsencrypt
+  # We change the owner to ocflets-encrypt before the renewal script runs
+  # Then we change it back after the renewal script runs
+  exec { 'chown-letsencrypt':
+    command => 'chown -R ocfletsencrypt:ssl-cert /var/lib/lets-encrypt/certs',
+    before  => Class['Ocf::Ssl::Default'],
+  }
+
+  exec {'chown-root':
+    command => 'chown -R root:ssl-cert /var/lib/lets-encrypt/certs',
+    require => Class['Ocf::Ssl::Default'],
+  }
+
+  # Change owner of /var/lib/lets-encrypt-certs to ocfletsencrypt
+  # postgres expects Class
```

Comment:
I think both of these comments are too far down? At least they don't appear to pertain to the line below here.